cmd/cmd.go

```diff
@@ -17,45 +17,30 @@ package cmd
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"os/signal"
 	"syscall"
 
 	"github.com/golang/glog"
 	multierror "github.com/hashicorp/go-multierror"
 	"github.com/spf13/cobra"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
 
 	proxyconfig "istio.io/api/proxy/v1/config"
 	"istio.io/pilot/model"
 	"istio.io/pilot/proxy"
+	"istio.io/pilot/tools/version"
 )
 
-const (
-	// ConfigMapKey is the key for mesh configuration data in the config map
-	ConfigMapKey = "mesh"
-
-	// DefaultConfigMapName is the default config map name that holds the mesh configuration.
-	DefaultConfigMapName = "istio"
-)
-
-// GetMeshConfig fetches configuration from a config map
-func GetMeshConfig(kube kubernetes.Interface, namespace, name string) (*proxyconfig.ProxyMeshConfig, error) {
-	config, err := kube.CoreV1().ConfigMaps(namespace).Get(name, v1.GetOptions{})
+// ReadMeshConfig gets mesh configuration from a config file
```

Comment:
Can you elaborate what you mean by update docker? Docker image won't function standalone.